backend/src/middleware/permissionsMiddleware.js

```diff
@@ -68,7 +68,11 @@ const isAuthor = rule({
 const isDeletingOwnAccount = rule({
   cache: 'no_cache',
 })(async (parent, args, context, info) => {
-  return context.user.id === args.id
+  if (isAdmin === true) {
+    return true
+  } else {
+    return context.user.id === args.id
```

Comment:
`context.user.id` should not be changed in my eyes.
Then the current user becomes the user to delete.
There must be an other solution for this.